homework-submission/todo-app-week1/src/components/ItemsComponent.js

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+
+const ItemsComponent = ({ items, done, action, addItem, inputRef }) => {
+  let lis = [];
```

Comment:
a one letter may not a big problem but it's not 💯 meaningful for everyone the `lis` name, right?
```suggestion
  let list = [];
```